packages/nameguard-react/src/components/Report/Banner.tsx

```diff
@@ -34,9 +34,9 @@ export function Banner({ report, parsedName }: Props) {
             {name}
```

Comment:
We should also be using a `DisplayedName` here, but we need to enhance the `DisplayedName` UI component for this to work correctly.

We need to buildENSName(name) to get an ENSName and then pass that into the `DisplayedName`.

Currently `DisplayedName` always displays the `displayName` field. But here we have a special case where we don't want to show the `displayName` field, we want to show the literal `name` field (which may or may not be normalized or unknown, etc..)

So we need to add another prop to `DisplayedName` that if `true` displays the `name` field rather than the `displayName` field no matter what the `normalization` is.

We need to do the exact same update inside `LoadingSkeleton`.